test/e2e/network/network_policy.go

```diff
@@ -1890,10 +2081,15 @@ func collectPodsAndNetworkPolicies(f *framework.Framework, podClient *v1.Pod) ([
 	return pods, policies, logs
 }
 
+type sctpPort struct {
+	port          int
+	readinessPort int
+}
+
 // Create a server pod with a listening container for each port in ports[].
 // Will also assign a pod label with key: "pod-name" and label set to the given podName for later use by the network
 // policy.
-func createServerPodAndService(f *framework.Framework, namespace *v1.Namespace, podName string, ports []int) (*v1.Pod, *v1.Service) {
+func createServerPodAndService(f *framework.Framework, namespace *v1.Namespace, podName string, ports []int, sctpPorts ...sctpPort) (*v1.Pod, *v1.Service) {
```

Comment:
change `ports` to `tcpPorts`

or have a single `ports` argument that takes a struct of port and protocol

Reply:
Changed. I'd prefer to keep the variadic function so `sctpPort`s are optional and do not impact existing tests.